substrate/frame/revive/rpc/build.rs

```diff
@@ -47,3 +51,40 @@ fn main() {
 	println!("cargo:rustc-env=TARGET={target}");
 	println!("cargo:rustc-env=GIT_REVISION={branch}-{id}");
 }
+
+fn build_ah_westend_wasm() {
+	let manifest_dir = env::var("CARGO_MANIFEST_DIR")
+		.expect("`CARGO_MANIFEST_DIR` is always set for `build.rs` files; qed");
+
+	let symlink_path = Path::new(&manifest_dir).join("asset_hub_westend_runtime.wasm");
+	let runtime_cargo_toml = Path::new(&manifest_dir)
+		.join("../../../../cumulus/parachains/runtimes/assets/asset-hub-westend/Cargo.toml")
+		.canonicalize()
+		.expect("Failed to resolve runtime path");
+	let wasm_path = Path::new(&env::var("CARGO_TARGET_DIR").unwrap_or_else(|_| {
+		Path::new(&manifest_dir)
+			.join("../../../../target")
+			.to_str()
+			.unwrap()
+			.to_string()
+	}))
+	.join(env::var("PROFILE").unwrap_or_else(|_| "debug".to_string()))
+	.join("wbuild/asset-hub-westend-runtime/asset_hub_westend_runtime.wasm");
+
+	substrate_wasm_builder::WasmBuilder::new()
+		.with_project(runtime_cargo_toml.to_str().expect("Invalid path"))
+		.unwrap()
+		.build();
```

Comment:
Why do we have to build it manually here? Can't we just add it as dependency?

Reply:
ah right I can access the generated  https://docs.rs/asset-hub-westend-runtime/0.35.0/asset_hub_westend_runtime/constant.WASM_BINARY.html directly

Reply:
I guess you have to make sure that this is not built with SKIP_WASM_BUILD then